core/test/services/iframe.spec.js

```diff
@@ -7,6 +7,7 @@ import { LuigiConfig } from '../../src/services/config';
 
 describe('Iframe', () => {
   let component;
```

Comment:
Wouldn't say so...
![image](https://user-images.githubusercontent.com/22825433/52043841-fc261680-2540-11e9-9871-3fa60c370758.png)
Don't trust VSCode's grey color, especially when `beforeEach` or `afterEach` are around ;)

Reply:
Ok, maybe I added a comment in a wrong line, because removing only this line will throw an error. But if you will also remove lines 13-19, it will be fine. `component` and `lastObj` variables are not used anywhere else later on, they are not needed.